htdocs/manage/settings/index.bml

```diff
@@ -445,6 +466,23 @@ body<=
     $ret .= $cats_with_settings{$given_cat}->{desc};
     $ret .= "</p></div>";
 
+    $ret .= "<div class='settings_content'>";
+    $ret .= "<div class='$given_cat'>";
+
+    if ( $given_cat eq "notifications" && $can_view_other ) {
+        # form for viewing another user's notifications
```

Comment:
In another ten years we'll probably be building sites to be projected directly into people's eyeballs. :)

I decided quite firmly that converting this page to TT was beyond the scope of what I wanted to do for this bug.